tests/server.test.js

```diff
@@ -32,7 +32,7 @@ describe('Signup', () => {
   //   name: 'Lucille',
   //   phoneNumber: 1555444333
   // }
-  // const validSignupData = 'name=%C3%89tienne&phone=1234567890'
+  const validSignupData = 'name=%C3%89tienne&phone=1234567890&household=roommates'
   const invalidPhoneData = 'name=%C3%89tienne&phone=012345678'
```

Comment:
```suggestion
  const invalidPhoneData = 'name=%C3%89tienne&phone=012345678&household=roommates'
```

Since we’re just testing the validity of phone data here, let’s make sure the rest of the payload is valid to minimize testing variables.